src/js/polyfills/console.js

```diff
@@ -241,3 +241,10 @@ Object.defineProperty(window, 'console', {
     writable: true,
     value: new Console()
 });
+
+Object.defineProperty(window, 'util', {
```

Comment:
Why is this here? If we need any specific utils we should expose them in the `tjs` namespace.

Reply:
Debug leftover. Though on second thought having all of util included makes sense IMHO, but another issue…